### PR TITLE
v0.5 config and requirements 

### LINKED ISSUE #188: Bump minimum k8s version to 1.15
Knative Serving 0.12 requires k8s 1.15+. riff should also require 1.15+ across the board.

### PR DESCRIPTION
**Pull request by jldec**
Suggest 5GB minikube/docker
Require k8s v1.15 (Fixes #188)

[preview](https://deploy-preview-198--projectriff.netlify.com/docs/v0.5/getting-started/minikube#install-minikube)

### CODE REVIEW

File: docs/v0.5/getting-started/gke.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ gcloud services enable \
 
 ## Create a GKE cluster
 
-Choose a new unique lowercase cluster name and create the cluster. For this demo, three nodes should be sufficient.
+Choose a new unique lowercase cluster name and create the cluster. For this demo, three nodes should be sufficient. v0.5 of riff requires Kubernetes v1.15 or later.
```
Comment (scothis) [inline docs/v0.5/getting-started/gke.md]: the gcloud command should be updated to create a compatible cluster. `--cluster-version=1.15` (`--cluster-version=latest` is still 1.13)

Comment (jldec) [reply docs/v0.5/getting-started/gke.md]: I just tried it and i got v1.15.8
Is is possible that it's not consistent across regions?

Comment (jldec) [reply docs/v0.5/getting-started/gke.md]: according to https://cloud.google.com/kubernetes-engine/versioning-and-upgrades

> `latest`: Specifies the highest supported Kubernetes version currently available on GKE in the cluster's zone or region.

`$ gcloud container get-server-config --zone $GCP_ZONE` will list the available versions for that zone.

Comment (scothis) [reply docs/v0.5/getting-started/gke.md]: yep, looks like `latest` is 1.15 now. I'm not quite sure what latest means as 1.13 is still the default, and even while 1.15 was available, latest would give you an older cluster.

Comment (trisberg) [reply docs/v0.5/getting-started/gke.md]: I always thought that `latest` worked well, gave me the latest listed in the UI which was always more recent than the default version

**Pull request merged**